docs/conversion/sakai_12_mysql_conversion.sql

```diff
@@ -618,3 +618,125 @@ DROP TABLE IF EXISTS SITEASSOC_CONTEXT_ASSOCIATIO;
+CREATE TABLE ASN_ASSIGNMENT (
+    ASSIGNMENT_ID                  VARCHAR(36) NOT NULL,
+    ALLOW_ATTACHMENTS              BIT,
+    ALLOW_PEER_ASSESSMENT          BIT,
+    AUTHOR                         VARCHAR(36),
+    CLOSE_DATE                     DATETIME,
+    CONTENT_REVIEW                 BIT,
+    CONTEXT                        VARCHAR(36) NOT NULL,
+    CREATED_DATE                   DATETIME NOT NULL,
+    MODIFIED_DATE                  DATETIME,
+    DELETED                        BIT,
+    DRAFT                          BIT NOT NULL,
+    DROP_DEAD_DATE                 DATETIME,
+    DUE_DATE                       DATETIME,
+    HIDE_DUE_DATE                  BIT,
+    HONOR_PLEDGE                   BIT,
+    INDIVIDUALLY_GRADED            BIT,
+    INSTRUCTIONS                   LONGTEXT,
+    IS_GROUP                       BIT,
+    MAX_GRADE_POINT                INT,
+    MODIFIER                       VARCHAR(36),
+    OPEN_DATE                      DATETIME,
+    PEER_ASSESSMENT_ANON_EVAL      BIT,
+    PEER_ASSESSMENT_INSTRUCTIONS   LONGTEXT,
+    PEER_ASSESSMENT_NUMBER_REVIEW  INT,
+    PEER_ASSESSMENT_PERIOD_DATE    DATETIME,
+    PEER_ASSESSMENT_STUDENT_REVIEW BIT,
+    POSITION                       INT,
+    RELEASE_GRADES                 BIT,
+    SCALE_FACTOR                   INT,
+    SECTION                        VARCHAR(255),
+    TITLE                          VARCHAR(255),
+    ACCESS_TYPE                    VARCHAR(255) NOT NULL,
+    GRADE_TYPE                     INT,
+    SUBMISSION_TYPE                INT,
+    VISIBLE_DATE                   DATETIME,
+    PRIMARY KEY(ASSIGNMENT_ID)
+);
+
+CREATE TABLE ASN_ASSIGNMENT_ATTACHMENTS (
+    ASSIGNMENT_ID VARCHAR(36) NOT NULL,
+    ATTACHMENT    VARCHAR(1024),
+    CONSTRAINT FK_HYK73OCKI8GWVM3AJF8LS08AC FOREIGN KEY(ASSIGNMENT_ID) REFERENCES ASN_ASSIGNMENT(ASSIGNMENT_ID),
+    INDEX FK_HYK73OCKI8GWVM3AJF8LS08AC(ASSIGNMENT_ID)
+);
+
+CREATE TABLE ASN_ASSIGNMENT_GROUPS (
+    ASSIGNMENT_ID VARCHAR(36) NOT NULL,
+    GROUP_ID      VARCHAR(255),
+    CONSTRAINT FK_8EWBXSPLKE3C487H0TJUJVTM FOREIGN KEY(ASSIGNMENT_ID) REFERENCES ASN_ASSIGNMENT(ASSIGNMENT_ID),
+    INDEX FK_8EWBXSPLKE3C487H0TJUJVTM(ASSIGNMENT_ID)
+);
+
+CREATE TABLE ASN_ASSIGNMENT_PROPERTIES (
+    ASSIGNMENT_ID VARCHAR(36) NOT NULL,
+    VALUE         LONGTEXT,
+    NAME          VARCHAR(255) NOT NULL,
+    PRIMARY KEY(ASSIGNMENT_ID,NAME),
+    CONSTRAINT FK_GDAT1B6UQIUI9MXDKTD6M5IG1 FOREIGN KEY(ASSIGNMENT_ID) REFERENCES ASN_ASSIGNMENT(ASSIGNMENT_ID)
+);
+
+CREATE TABLE ASN_SUBMISSION (
+    SUBMISSION_ID    VARCHAR(36) NOT NULL,
+    CREATED_DATE     DATETIME,
+    MODIFIED_DATE    DATETIME,
+    RETURNED_DATE    DATETIME,
+    SUBMITTED_DATE   DATETIME,
+    FACTOR           INT,
+    FEEDBACK_COMMENT LONGTEXT,
+    FEEDBACK_TEXT    LONGTEXT,
+    GRADE            VARCHAR(32),
+    GRADE_RELEASED   BIT,
+    GRADED           BIT,
+    GRADED_BY        VARCHAR(36),
+    GROUP_ID         VARCHAR(36),
```

Comment:
these 36s look too small

Reply:
GROUP_ID looks correct, these can't be created by users?